Program.markdown

```diff
@@ -80,39 +92,48 @@ For at kommunikasjonen skal tjene formålet med å koble sammen individer, må i
 
+Piratpartiet vil at borgere skal ha tilgang til all relevant informasjon for å kunne ta informerte beslutninger, og for å utføre nødvendig tilsyn med statsadministrasjonen. Uten åpenhet har vi ikke ekte demokrati. I fravær av demokrati kan vi ikke ta avgjørelser om offentlige ressurser, vi kan ikke hindre korrupsjon, og enkeltpersoner kan ikke holdes ansvarlig. Piratpartiet vil opprette en egen organisasjon for å bekjempe korrupsjon i offentlig sektor med rapportering til justisminister.
+
+Piratpartiet vil sikre låntakere mot verdifall i bolig, og la låntaker betale sitt boliglån ved å overføre eiendommen til banken. Utlån og avgifter skal være faste slik at låntakere kan endre sine låneinstitusjoner dersom de ønsker det, på samme måte som en bytter bank. Slik kan vi etablere et normalt forbrukermarked. Gjeldsordningsloven må forenkles med raskere sletting av gjeld. Folk skal ikke slås konkurs på grunn av økonomisk kollaps forårsaket av deres utlåner.
+
+Store bedrifter har fått altfor mye oppmerksomhet de siste årene. Bedrifter med færre enn 100 ansatte utgjør 99 % av norsk næringsliv. Over 1 million arbeider i små og mellomstore bedrifter. Piratpartiet ønsker å forbedre forholdene for små og mellomstore bedrifter med forenklet rammeverk. Piratpartiet vil se på mulighetene som internett- og delingsøkonomien gir.
+
+I den grad private aktører skal forvalte myndighet på vegne av det offentlige, må det stilles samme krav til offentligrettslige saksbehandlingsprinsipp og krav til god forvaltningsskikk.
+
 ### Piratpartiet vil konkret
 
 - Gjøre offentlige postjournaler tilgjengelig på Internett uten å måtte be om innsyn.
-- Gi alle individer uttalerett.
 - Publisere offentlige data med gjenbrukbar lisens, bare personvern og rikets sikkerhet kan stanse publisering.
 - Støtte «Holder de ord» og tilsvarende organisasjoner som registrerer hva de politiske partiene mener og hvordan de stemmer på Stortinget.
-- Utrede muligheten for å organisere saksbehandling på samme måte som feilhåndtering i programvare (mottak, klassifisering, behandling og vedtak/utbedring).
-- Innføre rådgivende nettbaserte avstemninger, Internett gjør det mulig å forenkle avstemninger uten uforholdsmessig stor ressursbruk.
+- Innføre rådgivende nettbaserte avstemninger i alle landets kommuner og fylkeskommuner. Internett gjør det mulig å forenkle avstemninger uten uforholdsmessig stor ressursbruk.
 - Kreve at digitale 3D-modeller vedlegges byggesøknader med ansvarsrett (tiltak uten ansvarsrett kan søkes uten 3D-modeller).
 - Kreve at kontrakter mellom det offentlige og private publiseres åpent.
```

Comment:
Riktig kopiert.
Forslag for LM2017: "Kreve at kontrakter mellom det offentlige og private offentliggjøres."